src/shared/vscode/commands2.ts

```diff
@@ -423,7 +452,7 @@ function getInstrumenter(
     if (!telemetryName && info?.startTime !== undefined && currentTime - info.startTime < threshold) {
         info.debounceCount += 1
         TelemetryDebounceInfo.instance.set(id, info)
-        getLogger().debug(`commands: skipped telemetry for "${id.id}" with key "${id.compositeKey}"`)
+        getLogger().debug(`commands: skipped telemetry for "${id.id}" with key "${JSON.stringify(id.compositeKey)}"`)
```

Comment:
`%O` is much more readable in the logs (unless you need to see deeply nested objects)
```suggestion
        getLogger().debug('commands: skipped telemetry for "%s" with key: %O', id.id, id.compositeKey)
```